tests/browser/cross-browser-compatibility.spec.ts

```
@@ -0,0 +1,210 @@
+    if (browserName === 'webkit') {
+      // Safari/WebKit might need special handling
+      console.log('Applying WebKit-specific configurations...')
+    }
+
+    // Run the core functionality test
+    const errors = await testCoreFunctionality(page, url)
+
+    // Create directory for screenshots if it doesn't exist
+    await page.evaluate(() => {
+      try {
+        require('fs').mkdirSync('./test-results/cross-browser', {
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)